editoast, gateway, osrdyne, flake: bump rust to 1.88

Sh099078 opened the pull request.

Khoyo committed:
editoast, gateway, osrdyne, flake: bump rust to 1.88

Signed-off-by: Younes Khoudli <younes.khoudli@epita.fr>

Khoyo reviewed (approved):
This looks eerily familiar

Khoyo committed:
editoast: fix clippy warning about if-let chains

Signed-off-by: Younes Khoudli <younes.khoudli@epita.fr>

Khoyo committed:
osrdyne: fix clippy warning about if-let chains

Signed-off-by: Younes Khoudli <younes.khoudli@epita.fr>

Khoyo committed:
gateway: fix clippy warning about if-let chains

Signed-off-by: Younes Khoudli <younes.khoudli@epita.fr>

Khoyo committed:
gateway: remove unused Endpoint struct in config

Was added when the gateway supported the datadog protocol, wasn't
removed at the same time

Signed-off-by: Younes Khoudli <younes.khoudli@epita.fr>

Khoyo committed:
gateway: use variables directly in the `format!` string

Signed-off-by: Younes Khoudli <younes.khoudli@epita.fr>

Khoyo committed:
osrdyne: use variables directly in the `format!` string

Signed-off-by: Younes Khoudli <younes.khoudli@epita.fr>

leovalais reviewed (approved):
weee
